src/data-reporting-quicksight-stack.ts

```diff
@@ -90,12 +90,12 @@ export class DataReportingQuickSightStack extends Stack {
     });
     interfaceCheckCR.node.addDependency(vPCConnectionResource);
 
-    const useTemplateArnCondition = new CfnCondition(
+    const useSpiceCondition = new CfnCondition(
       this,
-      'useTemplateArnCondition',
+      'useSpiceCondition',
       {
         expression:
-          Fn.conditionNot(Fn.conditionEquals(stackParams.quickSightTemplateArnParam.valueAsString, '')),
+          Fn.conditionEquals(stackParams.quickSightUseSpiceParam.valueAsString, 'yes'),
```

Comment:
Can we delete `quickSightTemplateArnParam` now?

Reply:
It is a backup method for v1.0. In fact, we have already stopped the publishing of this ARN.